articles/modules/ROOT/pages/migrating-explicit-lucene-indexes-to-native-schema-indexes.adoc

```diff
@@ -0,0 +1,102 @@
+= Migrating Explicit Lucene Indexes to Native Schema Indexes
+:slug: migrating-explicit-lucene-indexes-to-native-schema-indexes
+:author: Ali Maddahian
+:category: operations
+:tags: lucene, index, legacy, explicit, capacity, schema, full-text
+:neo4j-versions: 1.x,2.x,3.x,4.x
+
+Given that there are still some customers on older releases still utilizing legacy/explicit indexes, we will point out a few pointers here on how to convert these indexes to native schema indexes when upgrading to Neo4j version 4.x, as legacy/explicit indexes have been deprecated as of 3.5.x, and totally removed from the 4.x version.
+ 
+As to the background, Legacy/explicit indexes were the only type of indexes available prior to Neo4j 3.2(release date 2017).   These early version indexes which were implemented via Lucene under the cover resulted in significant write performance, and thus the need to replace them with the native schema indexex available in 3.3+ releases.   
+
+Aside from the performance perspective and equally important, explicit indexes also have to be explicitly/manually kept-up-to date when adding/removing/updating nodes/relationships/properties by way of the Java API (and/or stored procedures starting with 3.3.x), hence why they are called an explicit index.    
```

Comment:
"they are called an explicit index". Suggesting to align the complete phrase to one tense, i.e. singular "it is called an explicit index" or, plural: "they are called explicit indexes"